resources/js/core.js

```diff
@@ -134,9 +134,15 @@ const photoBooth = (function () {
         }
 
         loader.addClass('open');
-        public.startCountdown(config.cntdwn_time, $('#counter'), () => {
-            public.cheese(photoStyle);
-        });
+        if (nextCollageNumber) {
```

Comment:
Could be simplified with the following. 
```
 public.startCountdown(nextCollageNumber ? config.collage_cntdwn_time : config.cntdwn_time, $('#counter'), () => {
      public.cheese(photoStyle);
 });
```
Another approach would be to define a value `let countdownValue = nextCollageNumber ? config.collage_cntdwn_time : config.cntdwn_time` in the line above, which would increase the readability.